src-json/define.json

```diff
@@ -71,6 +71,11 @@
 		"define": "disable-hxb-cache",
 		"doc": "Use in-memory cache instead of hxb powered cache."
 	},
+	{
+		"name": "DisableHxbOptimizations",
+		"define": "disable-hxb-optimizations",
+		"doc": "Disable shortcuts hxb cache uses to speed up display requests."
```

Comment:
Somehow I had an aneurysm while reading this, though now that I know what it means it makes sense. Still, "Disable shortcuts used by hxb cache to speed up display requests." might make it easier to parse.